### ISSUE TITLE
Is XS tag still required when strandedness is given as a parameter?

### ISSUE THREAD

**Issue by weishwu** [gpertea/stringtie#450]
If I use `--fr` or `--rf` to specify the library strandedness, my understanding is that I then can use a BAM without XS tags. Am I right?

**Comment by unikill066** [comment]
@weishwu 
Yes, if you specify the library strandedness (--fr or --rf) in StringTie, you do not need XS tags in the BAM file. Also, do you know an answer to #467 ?

**Comment by unikill066** [comment]
High counts in Column 3 (forward) suggest a stranded forward library (e.g., Illumina's "fr-firststrand" protocol like TruSeq Stranded mRNA). And, high counts in Column 4 (reverse) indicate a stranded reverse library (e.g., dUTP-based protocols like TruSeq Stranded Total RNA).
